Structure_oM/Results/Global Results/StoreyDrift.cs

```diff
@@ -0,0 +1,64 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2024, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using BH.oM.Base;
+using BH.oM.Analytical.Results;
+using BH.oM.Quantities.Attributes;
+using System.ComponentModel;
+
+
+namespace BH.oM.Structure.Results
+{
+    [Description("Storey Drift (relative lateral displacement between consecutive floors) for a given Load Case or Load Combination.")]
+    public class StoreyDrift : StructuralGlobalResult, IImmutable, IResultItem
+    {
+        /***************************************************/
+        /**** Properties                                ****/
+        /***************************************************/
+
+        [Description("Storey name.")]
+        public virtual string Storey { get; }
+
+        [Description("Drift direction (direction of the storey lateral displacement).")]
+        public virtual string Direction { get; }
```

Comment:
```suggestion
        [Description("Direction of the storey lateral displacement.")]
        public virtual string Direction { get; }
```
Could this not be a `Vector`? Otherwise this could just be filled with any old string.

Reply:
@peterjamesnugent , description text just updated. Yes, ETABS returns a string ("X","Y" or "Z"), but we can turn it into a Vector. That will match the ETABS Global Reference System.

Reply:
@peterjamesnugent , thanks again for the comment.
Direction attribute now turned into a Vector Object as shown in latest commit b556ad92ddeab2490328aa34d33c79257727c066